### PR TITLE
Update to Plausible

### PR DESCRIPTION
**Pull request by avdata99**
Just an idea. Not tested

### CODE REVIEW

File: oki-wp-consent.php [hunk 1]
```diff
@@ -16,32 +16,8 @@
...
-      $analyticsTrackingID = (defined( 'OKI_GA_ID' ) ? OKI_GA_ID : '');
-    endif;
-
     ?>
-    <script>
-      var okiConsent = {
-        analyticsTrackingID: '<?php echo esc_attr( $analyticsTrackingID ); ?>'
-      };
-    </script>
+    <script defer data-domain="<?php get_bloginfo('url') ?>" src="https://plausible.io/js/plausible.js"></script>
```
Comment (amercader) [inline oki-wp-consent.php]: Not sure if `url` (eg https://blog.okfn.org) means host (eg blog.okfn.org) which is what we actually need.

**Comment by amercader** [comment]
@gocemitevski What is your take on this? 

The goal here is to stop using Google Analytics on the Wordpress sites and switch to plausible.io.
Is it worth creating a new version of this plugin that adds the necessary script tag, creating a new plugin all together or just edit the theme directly (see https://github.com/okfn/wordpress-theme/pull/23).
Ideally we could use the official [Plausible Wordpress plugin](https://wordpress.org/plugins/plausible-analytics/) but I'm not sure if our old version is supported or we would need to wait until the site is migrated

**Comment by gocemitevski** [comment]
@amercader it looks like the plugin is not very used. The last update was almost a year ago. However, this does not necessarily mean it's bad, but if it's just for the Plausible script, it might be a good idea to include it here in this library. However, I don't think the way the script is added in the MR is GDPR compliant. It needs to be loaded only after the user gives consent for being tracked. Maybe I'm missing something.

Another self-hosted open-source alternative to Google Analytics is https://matomo.org/. Just in case you're still looking for options.

**Comment by amercader** [comment]
@gocemitevski  so if I understand correctly you are saying that to just include a single `<script>` line which is what we want to do we can just do it in our custom plugin (this one) instead of installing the official one. That works for me.

With regards to GDPR compliance, unless they are lying here https://plausible.io/data-policy, I don't think we need to present a consent popup as we are not collecting Personal Identifiable Information or using cookies.

**Comment by gocemitevski** [comment]
> @gocemitevski so if I understand correctly you are saying that to just include a single `<script>` line which is what we want to do we can just do it in our custom plugin (this one) instead of installing the official one. That works for me.

Yep.

> With regards to GDPR compliance, unless they are lying here https://plausible.io/data-policy, I don't think we need to present a consent popup as we are not collecting Personal Identifiable Information or using cookies.

I think that the GDPR compliance here is specifically for https://plausible.io. It's a bit different with [oki-wp-consent](https://github.com/okfn/oki-wp-consent), since Plausible is the third-party. As far as I know, the user needs to give clear consent and approval for the Plausible scripts to be loaded. It must be clear to the user which data is shared with third-parties. For Google Analytics in [oki-wp-consent](https://github.com/okfn/oki-wp-consent) there are the Accept/Deny buttons... Accept loads the GA scripts, Deny does not and the user is not tracked in any way. That is how the consent makes a difference.

The implementation in this MR is not waiting for any confirmation from the user and it just always loads the Plausible script.

cc @amercader

**Comment by avdata99** [comment]
@amercader should we create a new `consent.js` file for plausible?
I found the `a` repo and the original consent file [here](https://github.com/okfn/a/blob/master/a/html/oki/consent/assets/js/consent.js)

This will be enough @gocemitevski?

**Comment by gocemitevski** [comment]
@avdata99 a new `consent.js` won't work since it has to include these scripts in runtime... below, where the initialization happens. You would probably want to extend the existing `consent.js`.

https://github.com/okfn/a/blob/master/a/html/oki/consent/assets/js/consent.js#L107

With the existing implementation these scripts are not loaded unless the user gives consent. As far as I know, this should not change and should apply for all third-party scripts that are loaded in `oki-wp-consent`.

cc @amercader

**Comment by amercader** [comment]
@gocemitevski Having looked at this with more detail I still think we don't need to ask for the user consent before loading the Plausible script. We (plausible) are not collecting any personal data or personally identifiable information (PII), nor setting any cookie at all. It's equivalent to loading any external JS script. We will clearly state the we use this in our Privacy Policy page, and we are not using CNAME cloaking so you can still block the script if you choose to do so.
So all in all I think it's a legitimate and ethical use to just load the Plausible script.

**Comment by gocemitevski** [comment]
@amercader feel free to merge this if OK.

**Pull request merged**